pyEpiabm/pyEpiabm/core/cell.py

```diff
@@ -87,6 +89,27 @@ def enqueue_person(self, person: Person):
         """
         self.person_queue.put(person)
 
+    def enqueue_testing(self, person: Person, test: str):
+        """Add person to testing queue for processing in testing
+         sweep.
+
+        Parameters
+        ----------
+        person : Person
+            Person to enqueue
+        test: str
+            Either 'PCR' or 'LFT' depending on the testing modality
+            to apply to that person.
+
+        """
+        if test == 'PCR':
+            self.PCR_queue.put(person)
+        elif test == 'LFT':
+            self.LFT_queue.put(person)
+        else:
+            raise ValueError("Testing modality specified is not allowed. " +
+                             "Should be 'PCR' or 'LFT'.")
+
```

Comment:
I would be happy with this being two separate methods - the functionality may diverge later and given there is no overlap right now it just seems messy to try to combine them